testsuite/features/secondary/trad_action_chain.feature

```diff
@@ -147,12 +142,10 @@ Feature: Action chain on traditional clients
     And I should see a "7. Reboot 1 system" text
 
   Scenario: Check that a different user cannot see the action chain for traditional client
-    Given I am authorized as "testing" with password "testing"
```

Comment:
This is no longer a valid scenario if we continue using admin

Reply:
Right! I did not read you on time, but testsuite results told me :)
Already fixed.